core/src/main/java/io/github/belgif/rest/guide/validator/core/model/PathItemDefinition.java

```diff
@@ -0,0 +1,17 @@
+package io.github.belgif.rest.guide.validator.core.model;
+
+import io.github.belgif.rest.guide.validator.core.parser.JsonPointer;
+import org.eclipse.microprofile.openapi.models.PathItem;
+
+public class PathItemDefinition extends OpenApiDefinition<PathItem>{
```

Comment:
some minimal class-level comments explaining PathItemDefinition and PathDefinition would be nice

Reply:
Done